docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Please add a link to the Sinkhorn class as {class}<code>~ott.solvers.linear.sinkhorn.Sinkhorn</code>.</p><p>Would also add {class}<code>problem &lt;ott.problems.linear.linear_problem.LinearProblem&gt;</code></p><p>and {class}<code>entropic map &lt;ott.problems.linear.potentials.EntropicMap&gt;</code> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>